libs/remix-ui/run-tab/src/lib/components/value.tsx

```diff
@@ -47,7 +47,7 @@ export function ValueUI (props: ValueProps) {
 
   return (
     <div className="udapp_crow">
-      <label className="udapp_settingsLabel" data-id="remixDRValueLabel">Value</label>
+      <label className="udapp_settingsLabel" data-id="remixDRValueLabel">Amount</label>
```

Comment:
I am not sure if this is a good idea as when we send tx using web3 libs, they call it `value` and even inside Solidity, it is accessed as `msg.value`

Reply:
Yes but it is confusing for users.  A user wants to know where to send some amount of wei or gwei or eth or whatever to a function.

Reply:
Sending "value" doesn't make immediate sense.